### DISCUSSION TITLE
Default value on a column ? (Sf5.4)

### DISCUSSION THREAD

**Discussion by toto33331** [symfony/symfony#45213] [Q&A] [answered]
Hi!
I wonder if there's a way to declare a default value for a column.
I don't look for how to modify a migration SQL or to put a default value on a form.
It seems the default option doesn't exist on the CLI when I create my entity with `php bin/console make:entity` but there's maybe an annotation I don't know to make my app "know" my default value...
**I want that when I do a `php bin/console make:migration`, the SQL code generated have to default value.**

Do anyone know how to do that ?

**Comment by toto33331** [comment, marked as answer]
Nevermind, I found out it's the option parameter `@ORM\Column(type="integer", options={"default":0})`

**Reply by stof** [reply]
side note: this question is not about Symfony at all, but about doctrine/orm, which is a separate project maintained by different people (at https://github.com/doctrine/orm)

**Reply by toto33331** [reply]
By knowing the answer I see it's doctrine/orm but when I asked, I didn't know

**Reply by stof** [reply]
Well, you were talking about your doctrine entities, and about the generated migrations. This might have given a choice between doctrine/orm and doctrine/migrations, but that's still some Doctrine parts.
